sagemaker_processing/fairness_and_explainability/fairness_and_explainability_byoc.ipynb

```diff
@@ -0,0 +1,1204 @@
+{
```

Comment:
<p>You can include the following cell to make it easy to find:</p><pre class="ql-syntax" spellcheck="false">! aws s3 cp {bias_report_output_path} ./ --recursive
</pre><p>and then open the file:</p><pre class="ql-syntax" spellcheck="false">import IPython
IPython.display.HTML(filename='bias_report.html')
</pre><p> </p><p>Change the file name with the correct one. I just made it up as an example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

Reply:
Thanks for the tip, I added the code snippet to the notebook and the html reports are displayed inline perfectly.